deploy/service_account.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: tower-resource-operator
+  namespace: tower-operator
```

Comment:
This is something I've been wondering about... do we need to fix this kind of thing into a namespace like this? I would think we'd want the flexibility  to put it in any namespace?

Reply:
Yeah, this is for deploying the ansible job operator to a specific non-default namespace.  Here is what we expect to deploy:
1. deploy the ansible job operator into one non-default namespace . e.g. `tower-operator`
2. create ansibleJob CR into default namespace.

By this way, we are able to demonstrate the ansible job operator is cluster scoped, it is watching all namespaces AnsibleJob CRs. and the AnsibleJob CR created outside fo the ansible job operator namespace will be able to launch its k8s runner jobs as expected.

The deploy yaml files are as sample. users can easily custmoize the namespace to their perferred ones. Also the image location has to be customized anyway.